ImplemenTriePrefixTree.java

```diff
@@ -0,0 +1,67 @@
+class Trie {
+
+    //Time Complexity : 0(n) where n:length of the word
+    //Space Complexity: 0(n)
```

Comment:
Please check the space complexity.

Reply:
It is correct as per me. Can you please point out the additional space I am using?

Reply:
To store one word of length l, it is `O(l)`. You have `n` words.

Reply:
Ok,
So the space complexity will be 0(ln)?